db/migrate/20240521150403_add_bulk_and_bulk_content_type_to_republishing_events.rb

```diff
@@ -1,7 +1,7 @@
 class AddBulkAndBulkContentTypeToRepublishingEvents < ActiveRecord::Migration[7.1]
```

Comment:
In the commit message, it should be "prevent accidentally setting `RepublishingEvent`s as `bulk: false`"

> This is an intermediate step before setting this back to have no `null` value to prevent accidentally setting `RepublishingEvent`s as `bulk` when they're not in future changes to the code.